format/spec.md

```diff
@@ -665,9 +665,34 @@ Table metadata consists of the following fields:
 | _optional_ | _required_ | **`sort-orders`**| A list of sort orders, stored as full sort order objects. |
 | _optional_ | _required_ | **`default-sort-order-id`**| Default sort order id of the table. Note that this could be used by writers, but is not used when reading because reads use the specs stored in manifest files. |
 |            | _optional_ | **`refs`** | A map of snapshot references. The map keys are the unique snapshot reference names in the table, and the map values are snapshot reference objects. There is always a `main` branch reference pointing to the `current-snapshot-id` even if the `refs` map is null. |
+| _optional_ | _optional_ | **`snapshot-statistics`** | A list (optional) of [table statistics](#table-statistics). |
```

Comment:
I'd still prefer `statistics` instead of `snapshot-statistics`. I can see how `table-statistics` is a bit too broad since they are based on a snapshot, but `snapshot-statistics` sounds more specific than needed.